libs/designsystem/src/lib/components/chart/chart-js/chart-js.service.ts

```diff
@@ -30,15 +48,101 @@ export class ChartJSService {
       dataLabels,
       customOptions,
       annotations,
+      datalabelOptions,
       highlightedElements,
     } = args;
 
-    const datasets = this.createDatasets(data, highlightedElements);
+    const suffix = datalabelOptions?.valueSuffix ? datalabelOptions.valueSuffix : '';
+
+    // We need to modify the datasets in order to add tooltips.
+    const datasets =
+      datalabelOptions?.showCurrent || datalabelOptions?.showMax || datalabelOptions?.showMin
+        ? this.createDatasets(this.addTooltips(data, datalabelOptions, suffix), highlightedElements)
+        : this.createDatasets(data, highlightedElements);
+
     const options = this.createOptionsObject({ type, customOptions, annotations });
-    const config = this.createConfigurationObject(type, datasets, options, dataLabels);
+    let config = this.createConfigurationObject(type, datasets, options, dataLabels);
+
+    if (type === ChartTypes.stock) {
+      config.options.plugins.tooltip.callbacks.label = (context) => {
+        return context.formattedValue + suffix;
+      };
+
+      config.options.scales.y.ticks.callback = (value) => {
+        return value + suffix;
+      };
+
+      // assuming that the first dataset controls the datespan.
```

Comment:
Is this an assumption we can count on?  
Anyways - any assumptions we have towards usage should be documented in the cookbook, such that the consumer of the component knows which assumptions we rely on 😄